www/gatsby-node.js

```diff
@@ -203,7 +203,7 @@ exports.createPages = ({ graphql, actions }) => {
 
       _.uniq(_.flatten(tagLists)).forEach(tag => {
         createPage({
-          path: `/blog/tags/${_.kebabCase(tag)}/`,
+          path: `/blog/tags/${_.kebabCase(tag.toLowerCase())}/`,
```

Comment:
If we don't lowercase first, we get something like `ja-mstack` and `jamstack` as two separate groups. 🙃 https://runkit.com/dschau/5b8e9aea035ca50012829b75